### ISSUE TITLE
all videos show "devicesupport" error

### ISSUE THREAD

**Issue by petibub** [KanoComputing/kano-video#50]
When going to "youtube" in kano-video, the video's show the typical youtube-error smiley with the message "www.youtube.com/devicesupport". 

According to that web page, youtube change their API and kano-video seems to be not compatible anymore. Any changes for an update?

**Comment by alex5imon** [comment]
@petibub thanks for reporting this problem. This is just a warning and it doesn't prevent you from watching videos. We are using an older API because it doesn't require authentication, so it is simpler for the user

**Comment by petibub** [comment]
Your answer suggests that it only happens on my kano (or I am doing something wrong). Thus, could you try to reproduce the problem? 

1) double click on the youtube app, wait until the yellow window opens.
2) Click on YOUTUBE. On my Kano, the white lower part of the window changes to:

YOUTUBE
10 videos
[         .   .            ] https://youtube.com/devicesupport
[          --             ] 7278K view - 3:56 min - by YouTube Help
[  err-msg here   ] https://youtube.com/devicesupport
[   YOUTUBE      ] http://m.youtube.com
                            WATCH | SAVE

Note that the video counter says: 10 videos, but only one row with a video is shown. Further note that when I hit WATCH, the video of "device is unsupported" is played in full-screen mode. Hitting Q brings you back to the problem. 

3) In the search bar, type "kano" and hit enter. On my kano, the white part of the windows changes to:

Showing results for "kano"
Results 1-10 of 1 video
[         .   .            ] https://youtube.com/devicesupport
[          --             ] 7278K view - 3:56 min - by YouTube Help
[  err-msg here   ] https://youtube.com/devicesupport
[   YOUTUBE      ] http://m.youtube.com
                            WATCH | SAVE
BACK (disabled)                     NEXT (enabled)

Note again the problem in the video counter. 

4) Hit WATCH to get the warning again. 
5) Hit NEXT to get more of the same results. On my Kano, they are all the same. 

This behavior does prevent me (and actually my daughter who loves playing with kano) from watching any youtube video on Kano. Can you reproduce that? If not, what am I doing wrong?

Thanks in advance

**Comment by alex5imon** [comment]
@petibub thanks for reporting. You are absolutely right, this has changed in the past few days. We will work in a fix as soon as possible

**Comment by petibub** [comment]
Cool! Thanks a lot in advance!

**Comment by RichMoly** [comment]
Yes, my kids get the same error (they like to watch Minecraft videos), so a solution would be great and warmly welcomed.

**Comment by the349** [comment]
# The API is no longer supported in kano_video/logic/youtube.py

`def search_youtube_by_keyword(keyword=None, popular=False, max_results=10,
                              start_index=1, parent_control=False):
    url = 'http://gdata.youtube.com/feeds/api/videos'
    params = {
        'v': 2,
        'vq': keyword,
        'racy': 'exclude',
        'orderby': 'relevance',
        'alt': 'json',
        'max-results': max_results,
        'start-index': start_index
    }`
